src/scss/billboard.scss

```diff
@@ -29,7 +29,7 @@
 .bb-chart-arc {
 	.bb-gauge-value {
 		fill: #000;
-		font-size: 12px;
+		font-size: 13px;
```

Comment:
the `.bb-chart-arc text` defines the `font-size:13px`, so rather than adding it need to be removed.
- https://github.com/naver/billboard.js/blob/master/src/scss/billboard.scss#L46